help/performance/capacity-planning.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Capacity Planning
+description: Learn about the steps necessary for capacity planning for  Adobe Commerce or Magento Open Source. 
+---
+
+## Capacity Planning
+
+we are considering single node/instance/machine setup and try to figure out above configuration
+capacity to handle traffic without break down. we are using universal formula to calculate server
+performance under limited resource & capability
+
+#### Website throughput (concurrent connection)
+
+***1) Lets figure out maximum user simultaneously access website for given configuration***
+
+**Example:**
+````
+Average PHP request time: 650ms
+CPU cores: 4
+Click frequency: 45 seconds
+
+4/0.65 = 3 page views per second * 60*0.75 = 135 max simultaneous user
+
+(Number of CPU cores / Average page Response Time in seconds) * 60 * User click
+frequency in second = maximum simultaneous users
```

Comment:
@vzabaznov 
As practically we can't determine max amount of simultaneous users at single point of time As depends upon server infrastructure 

it would be good idea to load test by using  performance tool kit and jmeter in-case to check capacity